### PR TITLE
0.55 upgrade

### PR DESCRIPTION
**Pull request by celaus**

### CODE REVIEW

File: pages/a/upgrade-your-cluster-to-0-55.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+[2016-07-22 11:58:12,931][ERROR][gateway.local.state.shards] [Brian Braddock]
+failed to read local state (started shards), exiting...
+org.elasticsearch.ElasticsearchException: unexpected field in shard state
+[index_uuid]
+...
+```
+
+## Settings Changes
+
+There have been several changes and depreciations to settings in 0.55, some due
```
Comment (andyellicott) [inline pages/a/upgrade-your-cluster-to-0-55.html]: Do you mean "deprecations" not "depreciations?"

File: pages/a/upgrade-your-cluster-to-0-55.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+backup](https://crate.io/a/backing-up-and-restoring-crate/) of your current
+tables so you can always return to what you had before. Furthermore be prepared
+to restart your entire cluster.
+
+## Index Changes
+
+Because of the Elasticsearch version upgrade, indices are not backwards
+compatible to 0.54. Be aware that once upgraded to 0.55, using the same index
+files in a 0.54 cluster will yield an error:
+
```
Comment (andyellicott) [inline pages/a/upgrade-your-cluster-to-0-55.html]: This section just explains a difference between .55 & .54, is there some action the customer should take? Rebuild indexes?

File: pages/a/upgrade-your-cluster-to-0-55.html (hunk 1 above)
Comment (andyellicott) [inline pages/a/upgrade-your-cluster-to-0-55.html]: Also..is there doc on the changes & deprecations? As a user, how do I learn whether the changes will affect me and what action I should take?

File: pages/a/upgrade-your-cluster-to-0-55.html [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+with the `CONF_FILE` environment variable, `-Des.config`, `-Des.default.config`,
+or `-Delasticsearch.config` parameters. You must call it _crate.yml_ and locate
+it in the _config_ directory, or another directory specified by the `CONF_DIR`
+environmental variable, or by the `-Des.path.home` parameter. The file must be a
+yaml file that ends in the _.yml_ extension.
+
+
+### Units Required
+
+You must now define units for time or byte based settings. Settings such as
```
Comment (andyellicott) [inline pages/a/upgrade-your-cluster-to-0-55.html]: 2nd sentence isn't a sentence. Change it to "This affects settings such as..."

File: pages/a/upgrade-your-cluster-to-0-55.html (hunk 2 above)
Comment (celaus) [inline pages/a/upgrade-your-cluster-to-0-55.html]: nope, all happens automatically. but the user should be aware that stuff happens under the hood.

**Pull request merged**